source/query-with-natural-language/enable-natural-language-querying.txt

```diff
@@ -82,7 +92,7 @@ Steps
          Once completed a :guilabel:`Logged in with Atlas account`
          message displays. 
 
-   .. step:: Enable the generative AI feature
+   .. step:: Read and Accept the Data Processing Agreement
```

Comment:
**For Tech Review**

Its been awhile since I enabled this setting / been prompted for the agreement, is this still true?

Reply:
We have replaced the consent modal with the disclaimer on the settings page. When a user clicks the "Enable AI Features" checkbox they agree with our terms to use GenAI.

Additionally, there is a link to the FAQ on the Atlas login card.

<img width="939" alt="Screenshot 2024-04-24 at 17 43 46" src="https://github.com/mongodb/docs-compass/assets/16307679/6f4f45f3-7594-4b46-a816-07aedb3a29fd">